cpu/fe310/Makefile.features

```diff
@@ -7,3 +7,7 @@ FEATURES_PROVIDED += periph_gpio periph_gpio_irq
 FEATURES_PROVIDED += periph_pm
 FEATURES_PROVIDED += periph_wdt
 FEATURES_PROVIDED += ssp
+
+ifneq (,$(filter periph_rtt,$(FEATURES_PROVIDED)))
+  FEATURES_PROVIDED += periph_rtc
```

Comment:
Can both features be used at the same time ?

Reply:
Probably not no

Reply:
Then I think one should also add a FEATURE_CONFLICTS variable somewhere.

Reply:
`FEATURES_CONFLICT += periph_rtt:periph_rtc` Should do the trick. Any good idea for adding the periph_rtc feature to fe310 boards offering the rtt feature?

I can also just add both to the fe310 cpu, I don't think it is practically possible to use the fe310 without 32.768 kHz crystal

Reply:
But `rtt_rtc` depends on `periph_rtt`